examples/snippets/low_latency/llm_augment_csv.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import tamr_toolbox as tbox
+
+
+# Initialize Tamr Client -- note port must be 9170 for low-latency match service
+client = tbox.utils.client.create(username="user", password="pw", host="localhost", port=9170)
```

Comment:
Can you rename this variable to `llm_client` to more clearly show that a standard `Client` will not work because of the port difference?